src/NearbyEntry.jsx

```diff
@@ -32,7 +32,7 @@ const Other = styled.div`
 `;
 
 const Heart = styled.span`
-  position: fixed;
+  position: absolute;
```

Comment:
Position absolute seems like an odd choice, but if it's working better for you, then sure. What are you doing with it? `Margin-left:300px` is a little drastic.